src/pages/LoginPage/LoginPage.page.jsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { useForm } from "react-hook-form";
+import { Button } from "../../StyledComponents/Button";
+import { Input } from "../../StyledComponents/Input";
+import { ButtonsContainer, ErrorMsg, FormContainer, InputsContainer } from "./Login.styles";
+
+export default function Login() {
+  const {
+    login,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+  const onFormSubmit = (data) => {
+    console.log(data);
+  };
+
+  return (
+    <FormContainer onSubmit={handleSubmit(onFormSubmit)}>
+      <div style={{ backgroundColor: "blue", height: "120px" }}></div>
+      <InputsContainer>
+        <label>
+          Nombre de Usuario
+          <Input
+            placeholder={"Nombre de Usuario"}
+            {...login("name", {
+              required: "Username is required"
+            })}
+          />
+          {errors.name && <ErrorMsg>{errors.name?.message}</ErrorMsg>}
+        </label>
+        <label>
+          Contraseña
+          <Input
+            placeholder={"Contraseña"}
+            type="password"
+            {...register("password", {
+              required: "Password is required",
+              pattern: {
+                message: "Incorrect Password",
+              },
+            })}
+          />
```

Comment:
Por ejemplo aqui:

Este input se le pasa el `register` para indicar que es un campo del formulario y se guardará en una variable `"password"`.

`required` se usa para especificar el error del mensaje cuando el campo no es llenado.

`pattern` es para validar que el input cumpla con una expresión regular, la expresión regular se especifica en `value` y `message` es el mensaje de error que se mostrará del formulario si falla esa validación.

Puedes importar la expresion regular `PASSWORD_STRENGTH_REGEX` para que no tengas que meterte tanto en expresiones regulares.

```suggestion
          <Input
            placeholder={"Contraseña"}
            type="password"
            {...register("password", {
              required: "Password is required",
              pattern: {
                value: PASSWORD_STRENGTH_REGEX,
                message: "Incorrect Password",
              },
            })}
          />
```